drivers/watchdog/wdt_esp32.c

```diff
@@ -168,12 +168,11 @@ static int wdt_esp32_init(const struct device *dev)
 
 	wdt_hal_init(&data->hal, config->wdt_inst, MWDT_TICK_PRESCALER, true);
 
-	int ret = esp_intr_alloc(config->irq_source,
-				ESP_PRIO_TO_FLAGS(config->irq_priority) |
-				ESP_INT_FLAGS_CHECK(config->irq_flags),
-				(ISR_HANDLER)wdt_esp32_isr,
-				(void *)dev,
-				NULL);
+	int ret = esp_intr_alloc(
+		config->irq_source,
+		ESP_PRIO_TO_FLAGS(config->irq_priority) |
+			ESP_INT_FLAGS_CHECK(config->irq_flags | ESP_INTR_FLAG_SHARED),
```

Comment:
same, arguabely the previous indentation was way better, I imagine you are using clang-format for this, sorry if you hoped that that'd save you from indentation review comments, it doesn't :-)